docs/pages/desktop-access/rbac.mdx

```diff
@@ -202,7 +202,7 @@ Windows users upon login.
 
 <Admonition type="warning" title="Local users only">
 Automatic user provisioning is only supported for local users, and does
-not take effect in Active Directory environments.
+not take effect in Active Directory environments. Windows hosts joined using Active directory are not supported. 
```

Comment:
Does the new text add information to "and does not take effect in Active Directory environments," or is this another way of saying the same thing? It might make sense to condense the text a bit:

```markdown
Automatic user provisioning is only supported for local users. Windows hosts
joined using Active Directory are not supported.
```

Reply:
I personally misinterpreted it- I read it as "will not create Active Directory users", not "will not work on Active Directory-joined hosts." Explicitly calling out that those hosts are not supported will prevent some confusion.

Reply:
I don't see how this is an improvement, but I may be biased since I wrote the original. What about the current phrasing isn't clear? User provisioning only works for local users, and local users don't apply in AD environments.

Reply:
@djohns7 Would you mind responding to Zac's comment?